spec/features/notifications_spec.rb

```diff
@@ -31,7 +31,7 @@
 
     it 'いいね！されたことが表示されること' do
       path = work_episode_checkin_path(checkin.work, checkin.episode, checkin)
-      expect(page).to have_link('チェックイン', href: path)
+      expect(page).to have_link('記録', href: path)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.